src/_data/sidenav.yml

```diff
@@ -1,38 +1,65 @@
-- title: Get started
+- header: Get started
+- title: Installation
   children:
     - title: Install Flutter
       permalink: /get-started/install
     - title: Test drive
       permalink: /get-started/test-drive
+
+
+- title: Learn Flutter
+  children:
+    - title: Introduction
+      permalink: /get-started/learn-flutter
+    - title: Dart language overview
+      permalink: https://dart.dev/overview
     - title: Write your first app
       permalink: /get-started/codelab
-    - title: Learn more
-      permalink: /get-started/learn-more
-    - divider
-    - title: Where to get Flutter support
-      permalink: /resources/support
-    - title: From another platform?
-      permalink: /get-started/flutter-for
+    - title: Fundamentals course
       children:
-        - title: Flutter for Android devs
-          permalink: /get-started/flutter-for/android-devs
-        - title: Flutter for SwiftUI devs
-          permalink: /get-started/flutter-for/swiftui-devs
-        - title: Flutter for UIKit devs
-          permalink: /get-started/flutter-for/uikit-devs
-        - title: Flutter for React Native devs
-          permalink: /get-started/flutter-for/react-native-devs
-        - title: Flutter for web devs
-          permalink: /get-started/flutter-for/web-devs
-        - title: Flutter for Xamarin.Forms devs
-          permalink: /get-started/flutter-for/xamarin-forms-devs
-        - divider
-        - title: Introduction to declarative UI
-          permalink: /get-started/flutter-for/declarative
-        - title: Flutter versus Swift concurrency
-          permalink: /get-started/flutter-for/dart-swift-concurrency
-    - title: "Dart language overview"
-      permalink: https://dart.dev/overview
+        - title: Introduction
+          permalink: /get-started/fundamentals-course
```

Comment:
I'd prefer if the path for these is just `/get-started/fundamentals` to keep paths short and simple. No need for the "course" qualifier.